apps/web/system/container.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'dry/system/container'
+require 'dry/system/hanami'
+
+# Web app container class for dependencies
+#
+# {http://dry-rb.org/gems/dry-system/ Dry-system documentation}
+module Web
+  class Container < Dry::System::Container
+    extend Dry::System::Hanami::Resolver
+
+    use :env
+
+    register 'vacancies.generators.rss' do
```

Comment:
it's not a good idea to create a new container for action. you can just put generator as `'web.vacancies.generators.rss'`. Could you change it?

Reply:
Sure, I changed it.